notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
<p>After talking it over with Shubham we thought that we should perhaps NOT offer a support email in this case (because the domain node may not exist, and we don't know if it has a network node, etc.), we should offer some general assistance (check your URL, contact the source where you obtained the URL) instead. If they are getting the URL incorrect that's probably the only feasible approach.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>